packages/cli/lib/lib/webpack/push-manifest.js

```diff
@@ -29,14 +31,14 @@ module.exports = class PushManifestPlugin {
 				'/': defaults
 			};
 
-			routes.forEach(filename => {
-				let path = filename.replace(/route-/, '/').replace(/\.chunk(\.\w+)?\.js$/, '').replace(/\/home/, '/');
-				manifest[path] = Object.assign({}, defaults, {
-					[filename]: {
-						type: 'script',
-						weight: 0.9
-					}
-				});
+			let path, css, obj;
+			scripts.forEach((filename, idx) => {
+				css = styles[idx];
+				obj = Object.assign({}, defaults);
+				obj[filename] = { type:'script', weight:0.9 };
+				if (css) obj[css] = { type:'style', weight:0.9 };
```

Comment:
This part _may_ be risky. In the event a route emits multiple `*.chunk.*.css` files, then the indexing won't align.

I'm not sure if this will happen, but I'm assuming it _might_ at some point~

Reply:
always expect the unexpected

Reply:
also can't we just leave the let inside the loop, this just looks ugly and I don't think this needs optimization

Reply:
Haha, it doesn't matter. No side effects in this loop, and multiple `let`s inside the loop looks ugly to me 😉

Reply:
so we should have let inside the loop for consistent code style? :p